apps/frontend/src/components/global/ExportCKLModal.vue

```diff
@@ -718,11 +720,34 @@ export default class ExportCKLModal extends Vue {
     for (const selected of this.selected) {
       this.addMetadataToPassthrough(selected);
       if ('evaluation' in selected) {
-        const result = validateChecklistMetadata(selected);
-        if (result.isError) {
-          SnackbarModule.failure(result.message);
+        // validate checklist metadata
+        const result = validateChecklistMetadata({
+          ...selected,
+          targetkey: null,
+          webordatabase: selected.webordatabase === 'true'
+        });
+        let errorMessages = [];
+        let isInvalid = false;
+        if (!result.ok) {
+          errorMessages.push(result.error.message);
+          isInvalid = true;
+        }
+
+        // validate metadata in each profile
+        for (const profile of selected.profiles) {
+          const result = validateProfileMetadata(profile);
+          if (!result.ok) {
+            isInvalid = true;
+            errorMessages.push(result.error.message);
+          }
+        }
+
+        // display error message upon any invalid inputs
+        if (isInvalid) {
+          SnackbarModule.failure(errorMessages.join('\n'));
```

Comment:
Can we do some more research to see if it's possible to restrict it to just integers natively?  Otherwise, I guess this is fine.

Reply:
it is now restricted by cancelling keyboard events if it's not a number character. It is still possible to get invalid keys in there, but the user would have to really try for that to happen. It would still fail upon export and give an appropriate snackbar error message if they did manage it.